### PR TITLE
Shrink Electron bundle by MBs!

### PR DESCRIPTION
**Pull request by feross**
Before: 127.6MB (48 MB zipped)
After: 120.5MB (42.8 MB zipped)

I accomplished this by enabling the ASAR archive format, and the (slightly gross) method of excluding files in `node_modules` that are likely to be unneeded.

Example: `test/`, `doc/`, etc.

### CODE REVIEW

File: bin/package.js [hunk 1]
```diff
@@ -109,11 +109,33 @@ var linux = {
...
+  electronPackager(Object.assign({}, all, darwin), done.bind(null, cb))
+}
+
+function buildWin32 (cb) {
+  electronPackager(Object.assign({}, all, win32), done.bind(null, cb))
+}
+
+function buildLinux (cb) {
+  electronPackager(Object.assign({}, all, linux), done.bind(null, cb))
+}
```
Comment (ngoldman) [inline bin/package.js]: d'oh, I was just working on making the packager accept specific target platforms.

Comment (feross) [inline bin/package.js]: Oh, sorry bout that! Let's chat about what we're working on on IRC!

File: package.json [hunk 2]
```diff
@@ -49,7 +49,8 @@
   "scripts": {
     "cleanup": "node ./bin/cleanup.js",
     "debug": "DEBUG=* electron .",
-    "package": "node ./bin/package.js",
+    "package": "npm prune && npm dedupe && node ./bin/package.js",
+    "size": "npm run package -- --darwin && du -ch dist/WebTorrent-darwin-x64 | grep total",
```
Comment (ngoldman) [inline package.json]: is this script just to get a file size?

File: bin/package.js (hunk 1 above)
Comment (ngoldman) [inline bin/package.js]: np I should have checked in :)

**Comment by ngoldman** [comment]
lgtm :+1:

File: package.json (hunk 2 above)
Comment (feross) [inline package.json]: Yep, exactly.

**Comment by feross** [comment]
Thanks for reviewing and merging. Nice new avatar, btw :+1:

**Comment by ngoldman** [comment]
Thanks! It's an old favicon I made that's been blown up, I still need to make a bigger version :laughing:

**Comment by feross** [comment]
Cool, the pixelated version looks cool too :)

**Pull request merged**